ext/bg/js/backend.js

```diff
@@ -150,7 +150,7 @@ class Backend {
             await profileConditionsDescriptorPromise;
 
             this.optionsSchema = await requestJson(chrome.runtime.getURL('/bg/data/options-schema.json'), 'GET');
-            this.defaultAnkiFieldTemplates = await requestText(chrome.runtime.getURL('/bg/data/default-anki-field-templates.handlebars'), 'GET');
+            this.defaultAnkiFieldTemplates = (await requestText(chrome.runtime.getURL('/bg/data/default-anki-field-templates.handlebars'), 'GET')).trim();
```

Comment:
Is this something that could cause issues in the future? I think it should work fine as long as you don't compare anything with the raw result from the `.handlebars` file. I did it this way because I've configured my editor to ensure a newline at the end of the file and it was added when I modified the `tags` template.

Reply:
It should probably be fine.